website/templates/emails/contributor_added_preprints_osf.html.mako

```diff
@@ -10,9 +10,9 @@
     <br>
     ${referrer_name + ' has added you' if referrer_name else 'You have been added'} as a contributor to the preprint "${node.title}" on the Open Science Framework: ${node.absolute_url}<br>
     <br>
-    You will ${'not receive ' if all_global_subscriptions_none else 'be automatically subscribed to '}notification emails for this project. To change your email notification preferences, visit your project or your user settings: ${settings.DOMAIN + "settings/notifications/"}<br>
+    You will ${'not receive ' if all_global_subscriptions_none else 'be automatically subscribed to '}notification emails for this preprint. To change your email notification preferences, visit your user settings: ${settings.DOMAIN + "settings/notifications/"}<br>
     <br>
-    If you are erroneously being associated with "${node.title}," then you may visit the project's "Contributors" page and remove yourself as a contributor.<br>
+    If you are erroneously being associated with "${node.title}," then you may visit the preprint and remove yourself as a contributor.<br>
```

Comment:
This and similar edits added to preprint emails to remove project language. You can't edit a project page to edit contributors any longer, fo rexample.